runbotics/docker-compose.yml

```diff
@@ -69,6 +69,7 @@ services:
       - RUNBOTICS_USERNAME=admin
       - RUNBOTICS_PASSWORD=admin
       - RUNBOTICS_HOT_RELOAD=true
+      # GUEST_BOT_ID: 691c15ee-0230-49c6-b794-9e33790652d3
```

Comment:
whats this?

Reply:
Guest bot it if you want to use guest bot for development.